docs/v1.md

```diff
@@ -175,25 +222,34 @@
 - hasDescription() : `boolean`
 - hasSummary() : `boolean`
 - id() : `string`
-- isApplicationPublishing() : `boolean`
-- isApplicationSubscribing() : `boolean`
-- isClientPublishing() : `boolean`
-- isClientSubscribing() : `boolean`
 - messages() : `Message[]`
 - servers() : `Server[]`
 - summary() : `string` | `undefined`
 - tags() : `Tags`
-- type() : `enum{'ClientSubscribing', 'ClientPublishing', 'ApplicationSubscribing', 'ApplicationPublishing}`
+- action() : `enum{'ClientSubscribing', 'ClientPublishing', 'ApplicationSubscribing', 'ApplicationPublishing}`
```

Comment:
👌🏼

Reply:
I think I would just leave `send` and `receive`, removing the others. WDYT @jonaslagoni @fmvilas ?

Reply:
Yup. I think parser DX should gracefully degrade in v2. Let's not make users choose method names depending on spec versions because they may not know in advance.